index.html

```diff
@@ -301,20 +324,42 @@ <h3>
     </div>
     <footer>
       <div class="contact">
-        <h2 class="about-me center-text">
-          Contact me
-        </h2>
+        <h2 class="about-me center-text">Contact me</h2>
         <p class="card-para center-text contact-p">
-          If you have an application you are interested in developing, 
-          a feature that you need built or a project that needs coding. I’d love to help with it 
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
         </p>
       </div>
-      <form action="/contact" method="POST">
-        <input type="text" id="username" name="username" Placeholder="Enter your name" value="Yeremias">
-        <input type="email" id="useremail" name="useremail" placeholder="enter your email" value="natajayan14@gmail.com">
-        <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here"></textarea>
+      <form action="https://formspree.io/f/xknkbwnn" method="POST">
+        <input
+          type="text"
+          id="username"
+          name="username"
+          placeholder="Enter your name"
+          required
+          minlength="30"
+        />
+        <input
+          type="email"
+          id="useremail"
+          name="useremail"
+          placeholder="enter your email"
+          required
+        />
+        <textarea
+          name="message"
+          id="message"
+          cols="30"
+          rows="10"
+          placeholder="Write your message here"
+          required
+          minlength="500"
```

Comment:
- [ ] Message input has `maxlength="500"` attribute not `minlength` as you know not everyone willing to write above 500 letter, so we restrict the input 500 to make the message short and to the point.